src/main/java/com/YouRL/controller/DemoController.java

```diff
@@ -0,0 +1,41 @@
+package com.YouRL.controller;
+
+import com.YouRL.service.DemoService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("api")
+public class DemoController {
```

Comment:
Should have more meaningful class name

Reply:
This is only for the demo. It will be changed in the future.